src/Symfony/Bundle/SecurityBundle/Resources/views/Collector/security.html.twig

```diff
@@ -346,16 +346,17 @@
                                             <td class="font-normal text-small">attribute {{ voter_detail['attributes'][0] }}</td>
                                             {% endif %}
                                             <td class="font-normal text-small">
-                                                {% if voter_detail['vote'] == constant('Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface::ACCESS_GRANTED') %}
+                                                {% if voter_detail['vote'].access == constant('Symfony\\Component\\Security\\Core\\Authorization\\Voter\\VoterInterface::ACCESS_GRANTED') %}
```

Comment:
Can use the isGranted() no?

Reply:
At this point, it's no more a Vote object unfortunately.
> Neither the property "isGranted" nor one of the methods "isGranted()", "getisGranted()"/"isisGranted()"/"hasisGranted()" or "__call()" exist and have public access in class "Symfony\Component\VarDumper\Cloner\Data".

Reply:
ha understood